internal-resources/git-workflow-and-contributor-guide.adoc

```diff
@@ -4,135 +4,226 @@
 :numbered:
 :internal-guide:
 
-= Stonesoup contributor guide
+= {Product_Name} Contributor Guide
 
-== Overview
+== How to use this guide
 
-This document describes the tasks needed to contribute to stonesoup documentation.
+The Contributor Guide is for writers on the {Product_Name} team, or anyone who needs to edit our documentation.
 
-IMPORTANT: If at *ANY TIME* you feel lost or don't know what to do, start asking questions on the _#forum-hac-docs_ slack channel.
+NOTE: To get help at any point, post in the _#forum-hac-docs_ Slack channel.
```

Comment:
Nitpick (very optional), but I think it's easier to click things at the end of the sentence. What do you think of:

`NOTE: To get help at any point, post in the Slack channel, #forum-hac-docs_`